docs/tutorial/kymotracking.rst

```diff
@@ -179,9 +179,9 @@ Computing the appropriate photons/pixel background considering a Poissonian nois
 
     offset = np.mean(kymo_cropped.green_image)
 
-The independently determined offset (in photons per pixel) can then be provided directly to `lk.refine_lines_gaussian`::
+The independently determined offset (in photons per pixel) can then be provided directly to `lk.refine_tracks_gaussian`::
 
-    refined = lk.refine_lines_gaussian(tracks, window=3, refine_missing_frames=True, overlap_strategy="skip", fixed_background=offset)
+    refined = lk.refine_tracks_gaussian(tracks, window=3, refine_missing_frames=True, overlap_strategy="skip", fixed_background=offset)
```

Comment:
The window here is the same as with `centroid`; yet in the current implementation, one is defined in physical units and one is not. Seems something is wrong?